samples/cellular/udp/src/main.c

```diff
@@ -126,6 +128,19 @@ static void lte_handler(const struct lte_lc_evt *const evt)
 	}
 }
 
+static bool is_nrf9160(void)
+{
+	char resp[32];
+
+	if (nrf_modem_at_cmd(resp, sizeof(resp), "AT+CGMM") == 0) {
+		if (strstr(resp, "nRF9160") != NULL) {
+			return true;
+		}
+	}
+
+	return false;
+}
```

Comment:
This sample only works on nRF 9 series SoCs. It may have had support for `qemu_cortex_m3` earlier, but that is no longer the case. I do not see why this function is necessary anymore, can we remove it?

Reply:
I'll update the sample accordingly then. I was under the assumption that qemu target was still required.